src/styles/global.css

```
@@ -0,0 +1,12 @@
+@import "tailwindcss";
+
+@theme {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@theme" (at-rule-no-unknown)

(at-rule-no-unknown)